### PR TITLE
Updated GSEA files to AGSEA files

### PR DESCRIPTION
**Pull request by cindyxio**
- I opened a new pull request because I went back and instead changed the `name` and file name in appyter.json back to GSEA, so there wouldn't need to be a new folder.

### CODE REVIEW

File: appyters/GSEA_Appyter/GSEA_Appyter.ipynb [hunk 1]
```diff
@@ -36,21 +38,21 @@
...
+    "#loading libraries\n",
     "import requests\n",
     "import urllib.request\n",
+    "from flask import url_for\n",
     "\n",
     "#performing GSEA\n",
     "import gseapy as gp\n",
     "from gseapy.gsea import GSEA\n",
+    "from maayanlab_bioinformatics.dge import limma_voom\n",
+    "from maayanlab_bioinformatics.dge.characteristic_direction import _chdir\n",
```
Comment (u8sand) [inline appyters/GSEA_Appyter/GSEA_Appyter.ipynb:50]: Please do not import private methods, if you need to access chdir directly I recommend using https://github.com/MaayanLab/geode

File: appyters/GSEA_Appyter/GSEA_Appyter.ipynb [hunk 2]
```diff
@@ -117,7 +119,7 @@
     "                default='GSE70466.gct',\n",
     "                examples={\n",
     "        'GSE70466.gct': 'https://appyters.maayanlab.cloud/storage/GSEA_Appyter/GSE70466.gct'},\n",
-    "                required=True,\n",
+    "                required='true',\n",
```
Comment (u8sand) [inline appyters/GSEA_Appyter/GSEA_Appyter.ipynb:122]: What's the reason for this change? Does `True` not work? In that case this looks like a bug in appyter.. "true" as a string seems like a hack

File: appyters/GSEA_Appyter/GSEA_Appyter.ipynb [hunk 3]
```diff
@@ -395,6 +439,8 @@
     "    num = round(num, 4)\n",
     "    if abs(num) < 0.01:\n",
     "        num = '{:.2E}'.format(Decimal(num)) #scientific notation\n",
+    "    if str(num) == '0.00E+2':\n",
```
Comment (u8sand) [inline appyters/GSEA_Appyter/GSEA_Appyter.ipynb:442]: I think it would be better to use:
```
if abs(num) <= 0.001:
  num = '<0.001'
elif abs(num) <= 0.01:
  num = '{:.2E}'.format(num)
```

File: appyters/GSEA_Appyter/appyter.json [hunk 4]
```diff
@@ -1,14 +1,22 @@
 {
   "$schema": "https://raw.githubusercontent.com/MaayanLab/appyter-catalog/master/schema/appyter-validator.json",
   "name": "GSEA_Appyter",
```
Comment (u8sand) [inline appyters/GSEA_Appyter/appyter.json:3]: The URL will be GSEA_Appyter.. we should change the directory & name if we wish it to be `AGSEA_Appyter`..

**Review by u8sand — CHANGES_REQUESTED** [review]

File: appyters/GSEA_Appyter/GSEA_Appyter.ipynb (hunk 2 above)
Comment (cindyxio) [reply appyters/GSEA_Appyter/GSEA_Appyter.ipynb:122]: Yes, this was something I was trying out, but neither `True` nor `'true'` so I ended up not using it, but I forgot to remove it. I think it may be because it is within a ChoiceField? I will remove it.

File: appyters/GSEA_Appyter/appyter.json (hunk 4 above)
Comment (cindyxio) [reply appyters/GSEA_Appyter/appyter.json:3]: Ah, ok. I was getting a version update earlier when I tried to change it to AGSEA_Appyter, though, so should the new version start again at 0.0.1?

File: appyters/GSEA_Appyter/GSEA_Appyter.ipynb (hunk 2 above)
Comment (u8sand) [reply appyters/GSEA_Appyter/GSEA_Appyter.ipynb:122]: @cindyxio I found the issue with required so if you leave it as True then it will work in the newer version of appyters when I release it

File: appyters/GSEA_Appyter/appyter.json (hunk 4 above)
Comment (u8sand) [reply appyters/GSEA_Appyter/appyter.json:3]: @cindyxio it should work as long as all files are in the new directory and no files are left in the old directory.
